apis/v1alpha1/gateway_types.go

```diff
@@ -85,8 +85,8 @@ type GatewaySpec struct {
 	//
 	// Support: Core
 	//
-	// +required
```

Comment:
Github won't let me comment on portions which are not diffed.
Above in this file, we have:
```
GatewayClassName string `json:"gatewayClassName"`
```
Does it make sense to have min and max length on that field?

Reply:
Updated, thanks!